options.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   tagline: 'Mic lint',
   eslint: eslint,
   eslintConfig: {
-    configFile: path.join(__dirname, 'eslintrc.json')
+    configFile: path.join(__dirname, 'eslint-config-mic/index.js')
```

Comment:
@danmakenoise just want to make sure this is working even though it's not a .json file?

Reply:
@ellell I tested locally and it seemed just fine! I'll do a triple check after publishing.